### DISCUSSION TITLE
Adding an official docke for Openradioss in DockerHub

### DISCUSSION THREAD

**Discussion by ntp3105** [OpenRadioss/OpenRadioss#2246] [Ideas]
I am wondering why Openradioss being an open-source, hasn't been added in dockerhub for easy installation?

**Comment by TessellateDataScience** [comment]
Perhaps not Docker, but it's been added to a similar environment [Apptainer](https://github.com/OpenRadioss/OpenRadioss/tree/main/Apptainer). We've personally got a Docker version also (hit me up if wanted), if you're perhaps concerned about HR having no clue about Apptainer but being all over Docker like a rash.

**Reply by QiangF** [reply]
@TessellateDataScience Do you have a link to your docker version? Thank you!

**Reply by TessellateDataScience** [reply]
Heya @QiangF, [short tutorial](https://github.com/orgs/OpenRadioss/discussions/4844#discussioncomment-17027286) to get you started includes link in video. How much experience do you have with running simulations?

Basically it's OpenRadioss (build) 2026-January (or then-abouts) on top of Ubuntu LTS:
`docker run -it --mount "type=bind,src=$pwd,target=/OpenRadioss/data" nchowlett/openradioss:0.3 sh`
